index.js

```diff
@@ -0,0 +1,65 @@
+let button1 = document.querySelector('#fourx');
+let button2 = document.querySelector('#thirtytwox');
+let button3 = document.querySelector('#img');
+let canvas = document.querySelector('#canvas');
+
+function draw1(arr) {
+    if (canvas.getContext) {
+        var ctx = canvas.getContext('2d');
+
+        arr.forEach((row, i) => {
+            row.forEach((column, j) => {
```

Comment:
@VikinGSpb It would be better to rename indexes to `rowIndex` or `rowIdx` instead of `i` and `j`